bin/secrets.sh

```diff
@@ -1,15 +1,29 @@
 #!/usr/bin/env bash
 
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-ANSIBLE_DIR="$( cd "$SCRIPT_DIR/../ansible" && pwd )"
+TOPLEVEL_DIR="$( cd "$SCRIPT_DIR/.." && pwd )"
 
-mkdir -p "$ANSIBLE_DIR/secrets"
+# Generates fresh zauth, TURN/restund, nginx/basic-auth and minio secrets as one-secret-per file. This can be useful in ansible-based deployments.
+# Then templates those secrets together in a secrets.yaml file for use in helm deployments.
+# USAGE:
+# ./bin/secrets.sh [ path-to-new-directory-for-secrets-output | default ./secrets_cache ]
 
-zrest="${ANSIBLE_DIR}/secrets/restund_zrest_secret.txt"
-zpub="${ANSIBLE_DIR}/secrets/zauth_public.txt"
-zpriv="${ANSIBLE_DIR}/secrets/zauth_private.txt"
-miniopub="${ANSIBLE_DIR}/secrets/minio_public.txt"
-miniopriv="${ANSIBLE_DIR}/secrets/minio_private.txt"
+OUTPUT_DIR="${1:-"$TOPLEVEL_DIR/secrets_cache"}"
+
+mkdir -p "$OUTPUT_DIR"
+
+zrest="${OUTPUT_DIR}/restund_zrest_secret.txt"
```

Comment:
I actually wrote the _exact same script_ yesterday https://github.com/wireapp/wire-server-deploy/pull/405/commits/8f9683fc4f846ed0b7f05f9fcb55e8a5690e294a

:) 

However I removed the writing to `.txt` files I wrote the secrets directly to the ansible inventory instead (through a `group_vars` file).

Perhaps that's also something we want to do here? Then you don't have to copy the secrets into your ansible inventory manually

Reply:
Ah well we could have saved some time by not doing that concurrently. Oh well :)

I took your ansible-inventory-generating code from the linked commit and added it here. Thanks! I also took the ZAUTH_CONTAINER idea and integrated it here.

As for the question of saving to text file: well, this PR also adds a basicAuth secret, which needs to be shared via group password manager and is neither used by helm nor by ansible. So at least that one needs its own file. Storing individual secrets in a file makes it easier to re-generate something or change the script. Do you think it's necessary/better to remove the writing-one-secret-to-file?